pkg/autoscaler/autoscaler.go

```diff
@@ -118,19 +86,6 @@ func (a *Autoscaler) Update(deciderSpec DeciderSpec) error {
 	return nil
 }
 
-// Record a data point.
-func (a *Autoscaler) Record(ctx context.Context, stat Stat) {
-	if stat.Time == nil {
-		logger := logging.FromContext(ctx)
-		logger.Errorf("Missing time from stat: %+v", stat)
-		return
-	}
-
-	// Proxied requests have been counted at the activator. Subtract
-	// AverageProxiedConcurrentRequests to avoid double counting.
-	a.buckets.Record(*stat.Time, stat.PodName, stat.AverageConcurrentRequests-stat.AverageProxiedConcurrentRequests)
-}
-
 // Scale calculates the desired scale based on current statistics given the current time.
```

Comment:
This was not done before, but it would be useful to understand what the return values are.